.env.dev

```diff
@@ -1,4 +1,5 @@
+# This loads for `yarn dev`, not the hosted dev site.
+# See https://github.com/mozilla/addons-code-manager#configuration
 REACT_APP_API_HOST=https://addons-dev.allizom.org
 # CDN URL
 PUBLIC_URL=https://addons-code-manager-dev-cdn.allizom.org
-REACT_APP_UNSAFE_ERROR_SIMULATION=true
```

Comment:
Why was this removed? Does it not do anything in a `yarn dev` environment?

Reply:
Correct. I put it in there mistakenly thinking I was setting a var for the dev site.